book/07-git-tools/sections/revision-selection.asc

```diff
@@ -38,7 +38,7 @@ Date:   Thu Dec 11 14:58:32 2008 -0800
     added some blame and merge stuff
 ----
 
-In this case, choose `1c002dd...`. If you `git show` that commit, the following commands are equivalent (assuming the shorter versions are unambiguous):
+In this case, say you're interested in the commit whose hash begins with `1c002dd...`. You can inspect that commit with any of the following variations of `git show` (assuming the shorter versions are unambiguous):
```

Comment:
Could you split these sentences onto two lines please?

Reply:
Damn, screwed that up, will fix.

Reply:
It was wrong to begin with. Thanks for fixing it up!

Reply:
Ah, you're right, so it was. Heh.